src/sagemaker/mxnet/estimator.py

```diff
@@ -99,8 +99,43 @@ def __init__(
                     * ``custom-image:latest``
 
             distributions (dict): A dictionary with information on how to run distributed
-                training (default: None). To have parameter servers launched for training,
-                set this value to be ``{'parameter_server': {'enabled': True}}``.
+                training (default: None). Currently we support distributed training with
+                parameter server and MPI [Horovod].
+                To enable parameter server use the following setup:
+
+                .. code:: python
+
+                    {
+                        'parameter_server':
+                        {
```

Comment:
(same for the other two)

```suggestion
                        'parameter_server': {
```

Reply:
Tensorflow estimator has same format
Why does the bracket have to be that way?
<img width="541" alt="Screen Shot 2020-07-22 at 12 17 55 PM" src="https://user-images.githubusercontent.com/10992635/88218796-68a14380-cc15-11ea-8816-73e5d131500b.png">